### PR TITLE
Config update

### PR DESCRIPTION
**Pull request by rubynouille**
Hi, I updated most dependencies, updated the readme.md with config and updated the version number (1.1.4) so it's just ready to go :)
Updated Nuxt to latest also
Fixed some import as well

### CODE REVIEW

File: package-lock.json
Comment (egidiusmengelberg) [inline package-lock.json:1]: Cool! looks good! Could you delete package-lock.json? That way only yarn is used and the package managers won't throw an error of multiple lockfiles

**Review by egidiusmengelberg — CHANGES_REQUESTED** [review]
Thanks! the rest looks very good.

File: package-lock.json
Comment (rubynouille) [reply package-lock.json:1]: Done! :)

**Review by egidiusmengelberg — APPROVED** [review]

**Comment by egidiusmengelberg** [comment]
Thanks! Merged it!

**Pull request merged**